README.md

```diff
@@ -1,7 +1,6 @@
 XenCenter
 =========
 
-[![Build Status](https://travis-ci.org/xenserver/xenadmin.svg?branch=master)](https://travis-ci.org/xenserver/xenadmin)

```

Comment:
GitHub also allows for action statuses to [be shown in the README.md](
https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge). I would switch to that to keep the label if possible, which some might find useful.